jobs_detector/main.py

```diff
@@ -26,7 +31,66 @@ def hacker_news(post_id, keywords, combinations):
     """
     # HINT: You will probably want to use the `BeautifulSoup` tool to
     # parse the HTML content of the website
-    pass
+
+    keywords = keywords.split(',')
+    combinations = combinations or [] # set this first so next line can iterate
+    combinations = [combinations.title() for combination in combinations]
+    main_page = requests.get(settings.BASE_URL.format(post_id))
+
+    # turn it into a BS4 object we can parse with html.
+    soupy = BeautifulSoup(main_page.text, 'html.parser')
```

Comment:
I'm not sure of this, but I think response.text strips the html tags off the response. We had a better time using response.contenxt, which keeps the HTML tags.